verifier/verifier.go

```diff
@@ -60,12 +60,12 @@ func Validate(proof shared.MerkleProof, labelHashFunc func(data []byte) []byte,
 }
 
 func asSortedSlice(s map[uint64]bool) []uint64 {
-	var ret []uint64
+	ret := make([]uint64, 0, len(s))
 	for key, value := range s {
 		if value {
 			ret = append(ret, key)
 		}
 	}
-	sort.Slice(ret, func(i, j int) bool { return ret[i] < ret[j] })
+	slices.Sort(ret)
 	return ret
 }
```

Comment:
The whole function can be written as:

```go
func asSortedSlice(s map[uint64]bool) []uint64 {
	ret := maps.Keys(s)
	slices.Sort(ret)
	return ret
}
```

if you add the `golang.org/x/exp/maps` dependency. Some of the functions in there have already been integrated into the standard library via the `maps` package, and it is likely that the others will eventually be part of the standard library as well 🙂

Reply:
I know, `maps.Keys` could also be used in different places. I didn't want to add a new dep in this PR though.